backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardMutationService.kt

```diff
@@ -115,6 +124,88 @@ class CardMutationService {
         }
     }
 
+    @GraphQLDescription("Creates a new digital entitlementcard from self-service portal")
+    fun createCardFromSelfService(
+        dfe: DataFetchingEnvironment,
+        project: String,
+        encodedCardInfo: String,
+        generateStaticCode: Boolean
+    ): CardCreationResultModel {
+        // TODO disable for production until the Koblenz project is ready to go live
+        val context = dfe.getContext<GraphQLContext>()
+        if (context.backendConfiguration.production) {
+            throw ForbiddenException()
+        }
+        if (project != KOBLENZ_PASS_PROJECT) {
+            throw NotKoblenzProjectException()
+        }
+
+        val cardInfo = parseEncodedCardInfo(encodedCardInfo)
+        val regionId = cardInfo.extensions.extensionRegion.regionId
```

Comment:
i don't think we can get the `regionId` from the `cardInfos` from the self service form since we have no user that is logged in and creates the card (There we normally get the `regionId`, because the regionManager or regionAdmin has a particular `regionId`)
You have to get it from `userentitlements` and enrich it.

Reply:
Ah, then I also need to update the user import a bit, currently it allows to upload the same user hash for different regions.
I thought we will pass the region through the form somehow.

Reply:
yes but you can also do it in a separate task

Reply:
Already done here